siliconcompiler/scheduler/taskscheduler.py

```
@@ -427,8 +427,12 @@ def __process_completed_nodes(self) -> bool:
                         # the next, so each node gets its own retry budget for now.
                         MPManager.get_path_cache().seed(
                             info["parent_pipe"].recv(), include_failures=False)
-                    except:  # noqa E722
-                        pass
+                    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)